docs/modding/suitmodding/addingblobs.md

```diff
@@ -0,0 +1,253 @@
+
+>Leave open your **GlacierKit** program
+
+:::info GlacierKit vs. RPKG
+While **RPKG** is helpful for viewing images in the **repository**, it’s best to work primarily in **GlacierKit**, as it will be your main modding tool. The **repository code lines** in both tools are identical for each file, so practicing in **GlacierKit** is encouraged for efficient modding.
+:::
+
+
+---
+## STEP 2: **FIND THE IMAGE**
+In this step we will have to look for the correct image code and find its **assembly path**.
+:::note
+You already did this step in the **GlacierKit** method but follow along
+:::
+   ### 1. Open your file
+Click on your suit and you should see its **repository** code on the right
+
+   ### 2. Locate the assembly path
+There will be a line of code starting with `"Image":"....`
+
+   For example:  
+   ```json
+   "Image": "images/unlockables_override/47_outfits_bloodmoney_gloves.jpg"
+   ```
+
+It consists of a
+- **file path** `images/unlockables_override/`
+- **file name** `47_outfits_bloodmoney_gloves.jpg`
+   
+   These 2 together are called the **assembly path** of your file's image.
+
+:::warning
+Keep **RPKG** and/or **GlacierKit** open, you’ll need to view the **assembly paths**.
+:::
+
+---
+
+## STEP 3: **SET UP A BLOBSFOLDER**
+In this step, you’ll create a custom folder structure that ***recreates*** the default image's **assembly path**.
+
+### 1. Make a blobsfolder
+
+   Create a new folder called ***`blobs`*** in your mod's root directory, where the manifest file is located. You can do this in **GlacierKit** or **File Explorer**.
+
+### 2. Recreate the file path
+
+    Create folders that match the **file path** of the image you want to replace.  
+    For example, if the **file path** is `images/unlockables_override/`, your `blobs` folder structure should look like this:
+   ```
+   blobs/images/unlockables_override/
+   ```
+### 3. Add your custom image  
+   - Place your image in the last folder you created (in this example, `unlockables_override`). 
+### 4. Recreate the file name
+   - Make sure the custom image has the exact same **file name** and **file type** as the original by changing it, for example:
+   ```
+   blobs/images/unlockables_override/47_outfits_bloodmoney_gloves.jpg
+   ```
+### 5. Confirm the assembly path
+This fully recreates the **assembly path** above of the image you're replacing.    
+   Your final folder structure should look like this:
+   ```
+   📁YourModFolder
+   ├── 📁blobs
+   │   └── 📁images
+   │       └── 📁unlockables_override
+   │           └── 47_outfits_bloodmoney_gloves.jpg
+   ├── 📁content
+   ├── manifest.json
+   └── project.json
+   ```
+:::danger Important
+- **Aspect Ratio**: Match the aspect ratio of your replacement image with the original to avoid distortion. 
+>*Example*: The Blood Money suit image has a 4:3 ratio (resolution 696x520), which you can check in **GlacierKit**.
```

Comment:
This can just be "for example", you don't really need so much emphasis and a separate quote block